packages/redux-routine/README.md

```diff
@@ -57,7 +57,7 @@ request has completed does the action creator procede to return the `SET_TEMPERA
 
 ## API
 
-### `createMiddleware( controls: ?Object )`
+### `createMiddleware( runtime: function )`
```

Comment:
Did this actually change? The function signature looks the same.

Reply:
I changed and reverted :P